debian/Dockerfile

```diff
@@ -23,6 +22,7 @@ RUN buildDeps=' \
 		autoconf \
 		automake \
 		bzip2 \
+		curl \
```

Comment:
Meh -- Alpine has it built into busybox, and it's generally useful, so I'm
+1 on keeping it (especially since we're dropping curl)